charts/templates/marble-injector.yaml

```diff
@@ -75,7 +75,7 @@ spec:
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: marble-injector
+  name: marble-injector-{{ .Release.Namespace }}
```

Comment:
Actually had this part in my local changes, missed pushing it
Thanks for catching :+1: